### ISSUE TITLE
[github.dev] All "Run" options for notebooks should lead to the "Continue Working On" prompt options

### ISSUE THREAD

**Issue by SiddhanthUnnithan** [microsoft/vscode#132057]
## TL;DR
Show the "Continue Working On" quick-pick options to users when invoking a "run" operation from within a notebook. 

## Overview
Currently, users are shown the following options to select kernels and run code in notebooks:
1. Select kernel prompt in the toolbar (right-hand-side).
2. Per-cell run button.

If the user does not have an extension installed which contributes kernels, they are shown a quick-pick prompt to "Install kernels from the marketplace". Clicking on this option will open the marketplace pane, filtered to show the Jupyter extension.

The Jupyter extension is currently unavailable in the web and will continue to be disabled until there is a clear run/execution story. Other extensions which contribute kernels (e.g., vscode-pyodide, kusto), are not Microsoft "official" extensions and thus cannot be shown as recommended extensions to install.

If the user does not have an extension installed which contributes kernels to use with a notebook, they should be shown quick-pick options equivalent to the "Continue Working On" button in the web, as an alternative to the current kernel installation prompt. Users will be given the option to graduate to desktop VS Code or a Codespace, both of which support environments in which the user can configure kernels and run their notebook cells.

We can think about a better graduation experience when moving from the web client to both desktop and codespaces, but that will be covered in a separate issue. For now we want both "upgrade" options to be shown to users instead of recommending to install kernels.

**Comment by greazer** [comment]
In summary, the suggestion here is that for the web scenario only, rather than pushing users to install the Jupyter and Python extensions in their web environment to perform an execution task (which won't work in the immediate term), push them directly to the "Continue Working On..." workflow.

**Comment by sergiobdrl** [comment]
This issue is the only result I found in the whole internet**s** with the words "install kernels from the marketplace". Turns out that since today I cannot run any of my notebooks on VSCode

Version: 1.59.1 (Universal)
Commit: 3866c3553be8b268c8a7f8c0482c0c0177aa8bfa
Date: 2021-08-19T11:53:52.479Z
Electron: 13.1.7
Chrome: 91.0.4472.124
Node.js: 14.16.0
V8: 9.1.269.36-electron.0
OS: Darwin arm64 20.3.0

I am prompted to install the kernels from the marketplace and taken to the extension page for Jupyter (@id:ms-toolsai.jupyter), which I already have installed. 

Please help!!!

**Comment by AdarshAnand67** [comment]
> the

Did you get the solution ??
I am facing a similar issue!
Kindly help, it has hampered my work a lot :(
